database/migrations/2024_09_18_140000_magento_bulk_requests_retry_of_field.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('magento_bulk_requests', function (Blueprint $table): void {
+            $table->unsignedBigInteger('retry_of')->after('id')->nullable();
```

Comment:
Would be nice if you could add a foreign key constraint that will set the value to NULL if the relation has been removed.